admin-policy.md

```diff
@@ -0,0 +1,75 @@
+  - The maintainer must not run in any election they maintain.
+- The voting system will not allow non-group members to vote; it may additionally deny some users the ability to vote if the user's account appears fraudulent.
+- The voting system will securely generate a v4 UUID for each user after they cast their first vote. When the user votes, their UUID will be stored along with their vote. When the election concludes, the UUID and vote will be published to allow auditing of the results.
+- Under this model (to be improved in the future), the maintainer is able to access the database linking votes to people, though the integrity of the results is relatively sound.
+
+## Removal of Admins
+
+Administrators may be removed by an unanimous vote among the other current admins. The current administrators may choose to hold a temporary election or wait until the next election if an administrator is voted out. 
+
+## Purpose of Administration
+
+- Lock/close posts (and remove when content policy specifies).
+- Ban users and deal with ban appeals (see the ban policy for more details).
+- Manage the group name/cover photo and its promotion.
+
+Note that there are many tasks such as Hackbot administration, Github projects, etc. that do not need to be solely managed by admins.
+
+## Deletion of Posts
+
+We believe that posts should only be deleted in a few specific cases. First of all, any spam from hacked accounts or bots will be deleted. Second, any posts violating Facebook's content policies will also be deleted. Finally, content can be deleted by an admin if the content is harassing a *specific* user, and that user requests that the content be removed. Note that generally objectionable content that does not target a *specific* user will not be removed (unless it violates Facebook's content policies), and will only be closed instead.
+
+All other content will be closed. The reason for this is to provide transparency as to what the admins deem to be bad content, and to leave examples of bad content in the group in an "archived" state so they can be used as examples of what not to post, as well as serving as precedent for closing future bad posts.
+
+## Ban Policy
+
+You may be banned by violating any of the rules defined in the content policy. A warning may be given first depending on the which policy was violated and the severity of the violation.
+
+When a user is banned, they have a chance to return to the group. They must make a post in Hackers
+Ban Appeals (not created yet), to argue why they shouldn’t be banned, or to apologize and request that they be re-added. If the user is not apologetic or we (the admins) and the community (jointly) do not agree with the user’s justification for their behavior, they will be banned from Hackers Ban Appeals and no further appealing can take place. Further policy violations in the main group after being re-added will result in immediate ban from both groups with no chance of appealing.
```

Comment:
>  we (the admins) and the community (jointly) do not agree with the user’s justification for their behavior

Would this also be put to a vote on the voting site?

Also, it might be better to write as "not apologetic or the admins and community jointly do not agree". Currently, the sentence is a little odd to read.